kami_test.go

```diff
@@ -146,10 +146,10 @@ func TestNotFound(t *testing.T) {
 	kami.NotFound(func(ctx context.Context, w http.ResponseWriter, r *http.Request) {
 		ok, _ := ctx.Value("ok").(bool)
 		if !ok {
-			w.WriteHeader(500)
+			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
-		w.WriteHeader(420)
+		w.WriteHeader(http.StatusNotFound)
```

Comment:
This should be something other than 404. We want this test to make sure that the default 404 handler isn't running. That's why I was returning a non-standard response code.